airflow/models/taskfail.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, ti):
             self.duration = int((self.end_date - self.start_date).total_seconds())
         else:
             self.duration = None
+        self.execution_date = None
```

Comment:
This attribute was replaced by `run_id` in this [PR](https://github.com/apache/airflow/pull/22260/files#diff-2311f4d526a373d3bfce455397245e2358a4fccccbfb2d132f78eae9e5f74fceL30-R35), and I don't think we need to recreate it. It is difficult to know if this can solve the issue without being able to reproduce it.

Reply:
I see, then I'll discard this PR.